src/administrator/components/com_kunena/install/install.script.php

```diff
@@ -17,11 +17,11 @@ class Com_KunenaInstallerScript
 {
 	protected $versions = array(
 		'PHP' => array (
-			'7.0' => '7.0.4',
+			'7.0' => '7.0.3',
 			'5.6' => '5.6.8',
 			'5.5' => '5.5.13',
 			'5.4' => '5.4.13',
-			'0' => '5.4.35' // Preferred version
```

Comment:
Although that sets a precedent on what the person should be running versus what is actually needed. Just an FYI most shared hosts where a majority of Kunena installs will be running on do not support PHP 7 yet. Many are pushing to 5.5 and 5.6. The preferred version should focus on two things: 1. the minimum version Kunena is absolutely required to run on 2. the End of Life conditions for older PHP versions. I believe on most frameworks PHP 5.5 is the minimum version required.